spec/firebase_app_distribution_helper_spec.rb

```diff
@@ -67,4 +74,14 @@
       expect(array).to eq(nil)
     end
   end
+
+  describe '#get_ios_app_id_from_archive' do
+    it 'returns identifier ' do
+      expect(CFPropertyList).to receive(:native_types)
+        .and_return(app_path)
+      expect(CFPropertyList).to receive(:native_types)
+        .and_return(identifier)
+      expect(helper.get_ios_app_id_from_archive("path")).to eq("identifier")
```

Comment:
I would like it if "path" was incorporated into the mocks, so that when reading the tests it is clear that the implementation of `get_ios_app_id_from_archive` actually uses it.